include/SDL_pen.h

```diff
@@ -0,0 +1,316 @@
+/*
+  Simple DirectMedia Layer
+  Copyright (C) 1997-2022 Sam Lantinga <slouken@libsdl.org>
+
+  This software is provided 'as-is', without any express or implied
+  warranty.  In no event will the authors be held liable for any damages
+  arising from the use of this software.
+
+  Permission is granted to anyone to use this software for any purpose,
+  including commercial applications, and to alter it and redistribute it
+  freely, subject to the following restrictions:
+
+  1. The origin of this software must not be misrepresented; you must not
+     claim that you wrote the original software. If you use this software
+     in a product, an acknowledgment in the product documentation would be
+     appreciated but is not required.
+  2. Altered source versions must be plainly marked as such, and must not be
+     misrepresented as being the original software.
+  3. This notice may not be removed or altered from any source distribution.
+*/
+
+/**
+ *  \file SDL_pen.h
+ *
+ *  Include file for SDL pen event handling.
+ *
+ *  This file describes operations for pressure-sensitive pen (stylus and/or eraser) handling, e.g., for input
+ *    and drawing tablets or suitably equipped mobile / tablet devices.
+ *
+ *  To get started with pens:
+ *  - Listen to ::SDL_PenMotionEvent and ::SDL_PenButtonEvent
+ *  - To avoid treating pen events as mouse events, ignore  ::SDL_MouseMotionEvent and ::SDL_MouseButtonEvent
+ *    whenever "which" == ::SDL_PEN_MOUSEID.
+ *
+ *  This header file describes advanced functionality that can be useful for managing user configuration
+ *    and understanding the capabilities of the attached pens.
+ *
+ *  We primarily identify pens by ::SDL_PenID.  The implementation makes a best effort to relate each :SDL_PenID
+ *    to the same physical device during a session.  Formerly valid ::SDL_PenID values remain valid
+ *    even if a device disappears.
+ *
+ *  For identifying pens across sessions, the API provides the type ::SDL_PenGUID .
+ */
+
+#ifndef SDL_pen_h_
+#define SDL_pen_h_
+
+#include "SDL_stdinc.h"
+#include "SDL_error.h"
+#include "SDL_joystick.h" /* For SDL_JoystickGUID */
+
+#include "begin_code.h"
+/* Set up for C function definitions, even when using C++ */
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct SDL_PenID {       /**< SDL_PenIDs identify pens uniquely within a session */
+    Uint32 id;
+} SDL_PenID;
```

Comment:
Why is this a struct instead of a Uint32 typedef?

Reply:
To improve type checking.
Anecdotally, it helped me find one bug.
Do you still want me to switch to a `typedef`?

Reply:
Yes please, for consistency with other similar types in SDL